### PR TITLE
Gp 2023.02.25

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 13/13_01_01_CIEL13_3B.txt [hunk 1]
```diff
@@ -1541,25 +1547,25 @@ So, if the child who is Roa were to die while Roa himself still lives, it would
 -- Page 228, Offset 33834.
 -- 「この世界はわずかな綻びがあれば、世界が世界自身のためにその綻びを修復します。
 "If ever a crack forms in the world,
- the world will patch it over in order to preserve itself.
+ the world will plaster it over in order to preserve itself.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_3B.txt:1550]: I think this could use a general rephrasing honestly. Structure and all. That said, I sort of came up with a more "generic" sounding metaphor, which I think is more generally understandable.
>In order to preserve itself, the world mends even the slightest of defects that appear in itself.

File: script/Ciel/Day 13/13_01_01_CIEL13_3B.txt [hunk 2]
```diff
@@ -1703,7 +1709,7 @@ That's why she wants to die――so she can finally be at peace.
 [sha:acbf891d879a80ccd318bc5f7903d0715015f0a7]{
 -- Page 10028, Offset 33864.
 -- 「違う……そんなのは、違う」
-// Assuming this is more of a 'you shouldn't kill yourself' than a 'no, you are incorrect'.
+// Assuming this is more of a 'you shouldn't kill yourself' / dying won't bring you peace than a 'no, you are incorrect'.
 "No... You're... making a mistake."
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_3B.txt:1713]: I still feel as though this one is hard to get a grasp on. The thing we have makes sense in context of the translation, so it works, but I get the feeling that it's not quite right. Well, it's beyond me as of now.

File: script/Ciel/Day 13/13_01_01_CIEL13_3B.txt [hunk 3]
```diff
@@ -1859,7 +1865,7 @@ For some reason, it's moving slowly.
 [sha:edecaba3a49fa06fc2884d7b928e17337fe04e8c]{
 -- Page 247, Offset 33894.
 -- 彼女なら―――あの代行者なら、刺されたと気付く間もない速度で突き刺せる筈なのに。
-She should―――Executor Ciel should be capable of finishing me off before I can even realize that I've been stabbed.
+She――Executor Ciel should be capable of finishing me off before I can even realize that I've been stabbed.
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_3B.txt:1868]: Think you need dashes on both sides.

>She――Executor Ciel――should...

**Review by yk1512 — APPROVED** [review]
no comments 🙏

File: script/Ciel/Day 13/13_01_01_CIEL13_3B.txt [hunk 4]
```diff
@@ -1221,7 +1225,8 @@ And you thought she loved you back. That's all.
 [sha:44f512b38c877f6791a2fc904e2dab8600d423bc]{
 -- Page 204, Offset 33774.
 -- 全部、全部ウソで塗りかためられた夢物語だった。
-All of it... Everything... It was nothing more than a fantasy.
+// Maybe improve 塗りかた phrasing here? Needs to sound not stupid
+All of it... Everything... It was nothing more than a fantasy founded on lies.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_01_CIEL13_3B.txt:1229]: Hmm, I feel like 'founded on' is basically just a more succinct version of 'built on a foundation of' anyway? Might prefer to keep it shorter here

File: script/Ciel/Day 13/13_01_01_CIEL13_3B.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_01_CIEL13_3B.txt:1550]: I dunno, I feel like that reads a bit dry, and the original doesn't strike me as hard to grasp

File: script/Ciel/Day 13/13_01_01_CIEL13_3B.txt [hunk 5]
```diff
@@ -747,7 +749,9 @@ It was close; Noel very nearly sent everything right down the drain.
 [sha:7f8f3ce8296ff11d92b31c936d0ab7eaf5e8d895]{
 -- Page 175, Offset 33686.
 -- ここまで周到に追い込んだのですから、完全に“起きて”もらうまでは泳がせてあげないと」
-I've been painstakingly driving you into a corner all this time, so I needed to let you run free until you fully awakened."
+// Maybe reword first half of this some more
+I'd spent all this time carefully driving you into a corner,
+ so I needed to let you run free until you fully awakened."
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_01_CIEL13_3B.txt:755]: For some reason that reads slightly awkwardly to me. Maybe replace with a different phrase?
> all this time getting you right where I wanted you,

File: script/Ciel/Day 13/13_01_01_CIEL13_3B.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 13/13_01_01_CIEL13_3B.txt:1230]: "founded on" is too abstracted and divorced from the metaphor. I don't really see why'd there'd be much of a concern for conciseness here either way, but if you want it shortened, I think the first part is more wordy than the latter.

>All of it——nothing more than fantasy given form by an edifice of lies.

Looking at this again, my second suggestion is honestly amazing, god I'm a genius.

File: script/Ciel/Day 13/13_01_01_CIEL13_3B.txt (hunk 1 above)
Comment (Petrikow) [reply script/Ciel/Day 13/13_01_01_CIEL13_3B.txt:1552]: It just feels a bit silly, is all. 綻び is a more common metaphor that English sort of lacks, so I don't think it was warranted thinking to much about that. 

That said, that last "itself" is a bit weird.
>In order to preserve itself, the world mends even the slightest of defects that appear in it.

File: script/Ciel/Day 13/13_01_01_CIEL13_3B.txt (hunk 5 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 13/13_01_01_CIEL13_3B.txt:755]: Sure
